### PR TITLE
chore: adjust text for earn and simplify active pool tap action

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Simplies the multipool card in the Discover Tab.

| iOS No Deposits Before | iOS No Deposits After | iOS With Deposits Before | iOS With Deposits After |
| ----- | ----- | ----- | ----- |
|  |  |  |  |

| Android No Deposits Before | Android No Deposits After | Android With Deposits Before | Android With Deposits After |
| ----- | ----- | ----- | ----- |
|  |  |  |  |

### Test plan

- [x] Tested locally on Android
- [x] Tested locally on iOS
- [x] Unit tests updated

### Related issues

- Fixes ACT-1333

### Backwards compatibility

Yes

### Network scalability

N/A

### CODE REVIEW

File: src/earn/EarnActivePools.tsx [hunk 1]
```diff
@@ -41,51 +41,34 @@ export default function EarnActivePools() {
     [localCurrencySymbol, totalSuppliedValue]
   )
 
+  function handleTap() {
+    AppAnalytics.track(EarnEvents.earn_active_pools_cta_press, {
```
Comment (finnian0826) [inline src/earn/EarnActivePools.tsx:45]: Does it make sense to continue using this even since the action will always be "myPools"?

Comment (MuckT) [reply src/earn/EarnActivePools.tsx]: Fixed in a1595f5e248e6cddf224e335bf8d7e5c2457e18b!

File: src/analytics/docs.ts [hunk 2]
```diff
@@ -606,7 +606,7 @@ export const eventDocs: Record<AnalyticsEventType, string> = {
   [EarnEvents.earn_withdraw_add_gas_press]: `When the user doesn't have enough for gas and clicks on the button to add gas token`,
   [EarnEvents.earn_info_learn_press]: `When the user taps 'Learn More' on the earn info page`,
   [EarnEvents.earn_info_earn_press]: `When the user taps 'Start Earning' on the earn info page `,
-  [EarnEvents.earn_active_pools_cta_press]: `When the user taps one of the CTAs in the active pools card in the discover tab. The action property denotes the CTA, either myPools or exploreOpenPools`,
+  [EarnEvents.earn_active_pools_cta_press]: `When the user taps one of the CTAs in the active pools card in the discover tab.`,
```
Comment (finnian0826) [inline src/analytics/docs.ts:609]:
```suggestion
  [EarnEvents.earn_active_pools_cta_press]: `When the user taps on the active pools card in the discover tab.`,
```

Comment (finnian0826) [reply src/analytics/docs.ts:609]: or maybe we just have a new event? it's not really a cta press anymore it's just active pool card press

Comment (MuckT) [reply src/analytics/docs.ts]: Fixed in 21174eee2f43974eae2b8ae98569fc42a6a858e6!

File: src/analytics/docs.ts [hunk 3]
```diff
@@ -606,7 +606,7 @@ export const eventDocs: Record<AnalyticsEventType, string> = {
   [EarnEvents.earn_withdraw_add_gas_press]: `When the user doesn't have enough for gas and clicks on the button to add gas token`,
   [EarnEvents.earn_info_learn_press]: `When the user taps 'Learn More' on the earn info page`,
   [EarnEvents.earn_info_earn_press]: `When the user taps 'Start Earning' on the earn info page `,
-  [EarnEvents.earn_active_pools_cta_press]: `When the user taps one of the CTAs in the active pools card in the discover tab. The action property denotes the CTA, either myPools or exploreOpenPools`,
```
Comment (finnian0826) [inline src/analytics/docs.ts:609]: Should this be moved down to the commented out deprecated events? (Not sure if we do that for all events we remove or just some.)

**Review by finnian0826 — APPROVED** [review]
One small comment but otherwise lgtm!

File: src/analytics/docs.ts (hunk 3 above)
Comment (MuckT) [reply src/analytics/docs.ts:609]: Added docs in fc89dd8637a7d339c9424e127a484ee140eec1ce!

**Pull request merged**